fenix/app/src/main/java/org/mozilla/fenix/shopping/middleware/ReviewQualityCheckTelemetryMiddleware.kt

```diff
@@ -19,7 +26,12 @@ private const val ACTION_DISABLED = "disabled"
 /**
  * Middleware that captures telemetry events for the review quality check feature.
  */
-class ReviewQualityCheckTelemetryMiddleware : ReviewQualityCheckMiddleware {
+class ReviewQualityCheckTelemetryMiddleware(
+    private val telemetryService: ReviewQualityCheckTelemetryService,
+    private val browserStore: BrowserStore,
+    private val appStore: AppStore,
+    private val scope: CoroutineScope,
+) : ReviewQualityCheckMiddleware {
```

Comment:
Considering that this middleware now has parameters, I think adding them to the doc above would be useful, similar to how the rest of the middlewares have:

```
 * @param telemetryService Service that handles telemetry events for review checker.
 * @param browserStore Used to record key when sending impression to avoid sending same impressions multiple times.
 * @param appStore The [AppStore] instance for dispatching [ShoppingAction]s.
 * @param scope The [CoroutineScope] to use for launching coroutines.
```

or something similar.

Reply:
Good catch, updated! Guess detekt doesn't catch when the docs don't have any param/property mentions but only catches when the docs have some of the params.